transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketChannelTest.java

```diff
@@ -151,6 +151,19 @@ public void testAutoReadOffDuringReadOnlyReadsOneTime() throws InterruptedExcept
         }
     }
 
+    @Test
+    public void testReadPendingIsNotResetAfterFirstRead() throws InterruptedException {
```

Comment:
@normanmaurer - Yes I think we can do that for all of the tests in this file which try different socket types. Let me take care as a followup PR?